OpenRA.Mods.Common/Widgets/ConfirmationDialogs.cs

```diff
@@ -88,11 +100,14 @@ public static class ConfirmationDialogs
 				};
 
 				if (!string.IsNullOrEmpty(otherText))
-					otherButton.GetText = () => otherText;
+				{
+					var otherTextMessage = modData.Translation.GetString(otherText);
+					otherButton.GetText = () => modData.Translation.GetString(otherTextMessage);
+				}
 			}
 		}
 
-		public static void TextInputPrompt(
+		public static void TextInputPrompt(ModData modData,
```

Comment:
We should also add the argument dicts to this prompt for symmetry with the other one and for future-proofing.

Reply:
No, I can't. There is a stylecop rule against unused arguments now.